### ISSUE TITLE
ESM version?

### ISSUE THREAD

**Issue by Mister-Hope** [ksloan/moment-mini#9]
Can you ship a esm version and add it through `exports`?

**Comment by ghiscoding** [comment]
~I created [moment-tiny](https://github.com/ghiscoding/moment-tiny) to address this since `moment-mini` doesn't seem to be maintained/updated anymore. I know MomentJS is deprecated but for a lot of us, we still need it so... As a side note, I wouldn't mind at all if the author of `moment-mini` wants to copy my implementation in here, it would be 1 less fork to support. Cheers~ (ended up deleting it since I switched to [`@formkit/tempo`](https://github.com/formkit/tempo) which is modern and has similarities with MomentJS)

**Comment by Jordan-Hall** [comment]
> I created [moment-tiny](https://github.com/ghiscoding/moment-tiny) to address this since `moment-mini` doesn't seem to be maintained/updated anymore. I know MomentJS is deprecated but for a lot of us, we still need it so... As a side note, I wouldn't mind at all if the author of `moment-mini` wants to copy my implementation in here, it would be 1 less fork to support. Cheers

Have you still got tiny as the link not working

**Comment by ghiscoding** [comment]
@Jordan-Hall no, I had some problems with it and it wasn't ESM at all and so I deleted it since no one was using it anyway. Instead I switched to [`@formkit/tempo`](https://github.com/formkit/tempo) which is full ESM, working great and has similarities with Moment (its parsing token are the same, which makes it easy to migrate) and my build size went down a lot :)
